File: 1877.数组中最大数对和的最小值/solution.cpp
class Solution {
public:
    int minPairSum(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int sz=nums.size();
        int ans=0;
        for (int i=0;i<sz/2;i++) {
            int a=nums.at(i),b=nums.at(sz-1-i);
            ans=max(ans,a+b);
        }
        return ans;
    }
};

File: 1877.数组中最大数对和的最小值/solution.go
func minPairSum(nums []int) int {
    sort.Ints(nums)
    ans:=0
    sz:=len(nums)
    for i:=0;i<sz/2;i++ {
        ans=max(ans,nums[i]+nums[sz-1-i])
    }
    return ans
}

func max(a,b int) int {
    if a>b {
        return a
    }
    return b
}